app/templates/components/public/side-menu.hbs

```diff
@@ -37,7 +37,7 @@
     {{t 'Schedule'}}
   {{/link-to}}
   {{#if shouldShowCallforSpeakers }}
-    {{#link-to 'public.cfs' class='item'}}
+    {{#link-to 'public.cfs' event.speakersCall.hash class='item'}}
```

Comment:
This is wrong. You shouldn't be putting the hash here. This would allow anyone to get the private cfs url by just going to the public event page.